packages/datagateway-common/src/dois/relatedIdentifiers.component.tsx

```diff
@@ -0,0 +1,445 @@
+    relatedIdentiferInputText,
+    dataCiteUrl
+  );
+
+  return (
+    <Paper
+      sx={{
+        background: (theme) =>
+          theme.palette.mode === 'dark'
+            ? theme.palette.grey[800]
+            : theme.palette.grey[100],
+        padding: 1,
+      }}
+      elevation={0}
+      variant="outlined"
+    >
+      <Grid container direction="row" spacing={1}>
+        <Grid item>
+          <Typography
+            variant="h6"
+            component="h4"
+            id="related-identifiers-label"
+          >
+            {t('DOIGenerationForm.related_identifiers')}
+          </Typography>
+        </Grid>
+        <Grid
+          container
+          item
+          spacing={1}
+          alignItems="center"
+          sx={{
+            marginBottom: relatedIdentifierError.length > 0 ? 2 : 0,
+          }}
+        >
+          <Grid item xs>
+            <TextField
+              label={t('DOIGenerationForm.related_identifier')}
+              fullWidth
+              error={relatedIdentifierError.length > 0}
+              helperText={
+                relatedIdentifierError.length > 0 ? relatedIdentifierError : ''
+              }
+              color="secondary"
+              sx={{
+                // this CSS makes it so that the helperText doesn't mess with the button alignment
+                '& .MuiFormHelperText-root': {
+                  position: 'absolute',
+                  bottom: '-1.5rem',
+                },
+              }}
+              InputProps={{
+                sx: {
+                  backgroundColor: 'background.default',
+                },
+              }}
+              value={relatedIdentiferInputText}
+              onChange={(event) => {
+                setRelatedIdentifierInputText(event.target.value);
+                setRelatedIdentifierError('');
+              }}
+              disabled={disabled}
+            />
+          </Grid>
+          <Grid container item spacing={1} xs="auto">
+            <Grid item>
+              <Button
+                variant="contained"
+                disabled={disabled}
+                onClick={() => {
+                  return checkDOI({ throwOnError: true })
+                    .then((response) => {
+                      // add DOI
+                      if (response.data) {
+                        changeRelatedIdentifiers((dois) => [
+                          ...dois,
+                          response.data,
+                        ]);
+                        setRelatedIdentifierInputText('');
+                      }
+                    })
+                    .catch(
+                      (
+                        error: AxiosError<{
+                          errors: {
+                            status: string;
+                            title: string;
+                          }[];
+                        }>
+                      ) => {
+                        // TODO: check this is the right message from the API
```

Comment:
Has this TODO been checked ?

Reply:
Just checked and yes that's the format for errors we get from DataCite so removed the TODO